src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py

```diff
@@ -65,6 +66,69 @@ def get_metadata(self) -> DeepDict:
         return metadata
 
 
+class PlexonLFPInterface(BaseLFPExtractorInterface):
+    """
+    Primary data interface class for converting Plexon LFP data.
+
+    Uses the :py:class:`~spikeinterface.extractors.PlexonRecordingExtractor`.
+    """
+
+    display_name = "Plexon LFP Recording"
+    associated_suffixes = (".plx",)
+    info = "Interface for Plexon low pass filtered data."
+    ExtractorName = "PlexonRecordingExtractor"
+
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        source_schema = super().get_source_schema()
+        source_schema["properties"]["file_path"]["description"] = "Path to the .plx file."
+        return source_schema
+
+    @validate_call
+    def __init__(
+        self,
+        file_path: FilePath,
+        verbose: bool = False,
+        es_key: str = "ElectricalSeries",
+        stream_name: str = "FPl-Low Pass Filtered",
```

Comment:
I don't think we usually have spaces in these names, do we? Matnwb can sometimes run into problems with groups have spaces in their names